deployement/fix-mapping/FIX.md

```diff
@@ -0,0 +1,29 @@
+# Mappingservice fix for LifeCycle
+Mappingservice chokes in Harmonizations table copy.
+
+Sources:
+https://drive.google.com/drive/u/0/folders/1LgShQvLN8GHTLZv-xvQvGBbfoW7zN3W1
+
+## Prerequisites
+- Upload the new “HarmonizationsShadow” to production
+- Update mapping target entity to LifeCycle_HarmonizationsShadow 1x
+- Update mapping entity targets to LifeCycle_HarmonizationsShadow 22x
+
+## Steps to run the mapping
+- Remove “harmonizations” attribute reference from “LifeCycle_CoreVariables” in the python script
```

Comment:
suggest to change to; Remove “harmonizations” attribute reference from “LifeCycle_CoreVariables” by running script [name-of-script] + link to script

Reply:
now i see the jobs png , maybr we could inline in the md file